docs/spec.md

```diff
@@ -6317,33 +6317,47 @@ def baseline_type(x: Value | Placeholder | Type) -> Type:
 floating-point tensor types. This happens via converting quantized elements
 which represent integer values of the storage type into corresponding
 floating-point values of the expressed type using the zero point and scale
-associated with the quantized element type. At the moment, this function only
-works for per-tensor quantization. Per-axis quantization is work in progress
-([#1574](https://github.com/openxla/stablehlo/issues/1574)).
+associated with the quantized element type.
 
 ```python
+def get_zero_points(quantized_type, result_type):
```

Comment:
Let's rename to `compute_zero_points` and `compute_scales`? `get_` suggests that this is a getter, whereas there's a bit more going on than just getting.

Reply:
done. Thanks!